examples/hydration-with-plugins/index.tsx

```diff
@@ -19,9 +19,7 @@ const myHistory = createBrowserHistory();
 const appRoutes = [homeRoute];
 
 const getStateFromServer = async () => {
```

Comment:
Can we rename this set of examples to `examples/hydration`, not `examples/hydration-with-plugins`

Reply:
done